### PR TITLE
Simple speed up for CIPLabeler

### PR DESCRIPTION
**Pull request by d-b-w**
#### What does this implement/fix? Explain your changes.
std::list is almost never the data structure that you want. vector uses fewer allocations, and the data has better locality. This speeds up the CIPLabeler on some protein examples by about 2x (75s to 38s)

### CODE REVIEW

File: Code/GraphMol/CIPLabeler/Digraph.cpp [hunk 1]
```diff
@@ -74,9 +75,12 @@ int Digraph::getNumNodes() const { return d_nodes.size(); }
 
 std::vector<Node *> Digraph::getNodes(Atom *atom) const {
   std::vector<Node *> result;
-  auto queue = std::list<Node *>({getCurrentRoot()});
+  std::deque<Node*> queue = {getCurrentRoot()};
```
Comment (greglandrum) [inline Code/GraphMol/CIPLabeler/Digraph.cpp:78]: I completely agree that `std::list` is inappropriate here, but wouldn't it be a bit faster to just use a `std::vector` for queue and do a loop `for(auto idx=0u; idx<queue.size(); ++idx)` ?

Comment (greglandrum) [reply Code/GraphMol/CIPLabeler/Digraph.cpp:78]: I tried this with a bunch of ChEMBL molecules and it doesn't make any difference, but maybe you could try with the proteins you were looking at?

Comment (d-b-w) [reply Code/GraphMol/CIPLabeler/Digraph.cpp:78]: good call!

**Review by greglandrum — APPROVED** [review]
LGTM

**Comment by greglandrum** [comment]
I'm holding off on merging this until after the release is done at the end of the week.

**Pull request merged**